src/router/index.js

```diff
@@ -55,6 +57,19 @@ export default (store) => {
         name: 'onboarding',
         path: '/onboarding',
         component: Onboarding
+      }, {
+        name: 'new-account-intro',
+        path: '/new-account-intro',
+        component: NewAccountIntro
+      }, {
+        name: 'new-account-create',
+        path: '/new-account-create',
+        component: NewAccountCreate
+      }, {
+        name: 'new-account-confirmation',
+        path: '/new-account-confirmation',
+        component: NewAccountConfirmation,
+        props: true
```

Comment:
`props: true` makes the component to receive route params in properties, but currently this feature is not used.